### ISSUE TITLE
[BUG] Legacy Tool Library Setting Not Working Anymore

### ISSUE THREAD

**Issue by LinnZawWin** [MscrmTools/XrmToolBox#1304]
**Describe the bug**
Even though the "Use legacy tool library" setting is enabled, only the new tool library is loading.

**To Reproduce**
Set "Use legacy tool library" setting to True in the Settings.
Open the Tool Library under Configuration menu.
The new tool library is opened as a new tab.

Close and reopen the XrmToolBox
Open the Tool Library under Configuration menu.
The new tool library is opened as a new tab.

**Expected behavior**
Legacy tool library is opened instead of the new tool library.

**Desktop (please complete the following information):**
 - XrmToolbox Version 1.2023.12.68

**Comment by MscrmTools** [comment]
You are correct, I forgot to remove this settings and the legacy tool library is no more available at all
